Makefile

```diff
@@ -46,12 +46,15 @@ install-service: check-prev-proxies
 
 	install -D -m 755 trigger-pac4cli $(DESTDIR)/etc/NetworkManager/dispatcher.d/trigger-pac4cli
 	install -D -m 755 pac4cli.sh $(DESTDIR)/etc/profile.d/pac4cli-proxy.sh
+	install -D -m 644 pac4cli.config $(DESTDIR)/etc/pac4cli/pac4cli.config
 
 install-bin:
 	install -D -m 755 main.py $(DESTDIR)$(bindir)/pac4cli
 	@sed -i -e '1s+@PYTHON@+'$(PYTHON)'+' $(DESTDIR)$(bindir)/pac4cli
 
 	install -D -m 644 pac4cli.py $(DESTDIR)$(pythonsitedir)/pac4cli.py
+	install -D -m 644 wpad.py $(DESTDIR)$(pythonsitedir)/wpad.py
+	install -D -m 644 servicemanager.py $(DESTDIR)$(pythonsitedir)/servicemanager.py
```

Comment:
I agree. This is not a design decision yet. It's still too early to me. I'm still in the early stages of investigation/learning phase in adapting the project to mac os.

Ideally I shouldn't ask for a pull request except after finishing the feature and having a final design. But since my knowledge in Python is still pretty limited, I opted to ask you to review the changes in small chunks and fail early.

As a matter of fact, I was wondering how to install these helpers scripts as a *private* site-packages for this project only (vs. public for all). But this is a question I'll seek an answer to it later.